config/config.sample.php

```diff
@@ -9,5 +9,9 @@
         'markdown' => [
             'engine' => 'engine', // See https://twig.symfony.com/doc/3.x/filters/markdown_to_html.html and then set to `new DefaultMarkdown()`
         ],
+        'context' => [
+            // Name of Twig template variable name that contains View instance
+            'view' => '_view',
```

Comment:
The key name could be `viewVar`. Could we use default var name `this` instead of `_view`?

Reply:
If we're changing the configuration, then backwards compatibility is gone. I'd be open to changing that default name then ... however, we're making it rough to transition anything large here.

Reply:
We are making the var name is configurable so anyone can set it to old name if needed.

Reply:
I know ... but will they realize why? I guess we can explain the configuration change at the top of the readme.